reference/opt/shaders-msl/vulkan/vert/multiview.multiview.nocompat.vk.vert

```diff
@@ -0,0 +1,32 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct MVPs
+{
+    float4x4 MVP[2];
+};
+
+struct main0_out
+{
+    float4 gl_Position [[position]];
+    uint gl_Layer [[render_target_array_index]];
+};
+
+struct main0_in
+{
+    float4 Position [[attribute(0)]];
+};
+
+vertex main0_out main0(main0_in in [[stage_in]], constant uint* spvViewMask [[buffer(24)]], constant MVPs& _19 [[buffer(0)]], uint gl_InstanceIndex [[instance_id]])
+{
+    main0_out out = {};
+    uint spvMaxViewIndex = 31 - clz(*spvViewMask);
```

Comment:
I think it makes more sense to pass an array of view indices here rather than clz which can have gaps. I expect something like:
```
uint spvMaxView = spvViewMask[0];
uint gl_ViewIndex = spvViewMask[InstanceIndex % spvMaxViewIndex + 1];
```

Reply:
The view mask is fixed to the render pass, so it should be possible to make this constant data rather than loading it from a buffer and having to manage that from the API side.

Reply:
Done.